C++/13-Bit Manipulation/11-Power_of_two.cpp

```diff
@@ -0,0 +1,22 @@
+  #include<bits/stdc++.h>
```

Comment:
Is this related to `Bits` or `DSA`? I guess this one should be in the programming fundamentals. What do you think?